infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -162,6 +161,17 @@ public ManagementServer(ServerContext serverContext) {
         }
     }
 
+    private void bootstrapPrimarySequencerServer() {
+        try {
+            String primarySequencer = latestLayout.getSequencers().get(0);
+            getCorfuRuntime().getRouter(primarySequencer).getClient(SequencerClient.class)
```

Comment:
We should log the acks/nacks for the bootstrap, as more than one could be bootstrapping at the same time.